test/integration/puppetlabs/services/master/tasks_int_test.clj

```diff
@@ -0,0 +1,101 @@
+(ns puppetlabs.services.master.tasks-int-test
+  (:require [clojure.test :refer :all]
```

Comment:
oh lord, there's both puppetlabs.puppetserver.bootstrap-testutils and puppetlabs.trapperkeeper.testutils.bootstrap, that confused me momentarily